internal/changeTracking/command_create.go

```diff
@@ -0,0 +1,282 @@
+newrelic changeTracking create \
+  --entitySearch "id = '<Entity GUID>'" \
+  --category Operational \
+  --type "Maintenance Window" \
+  --description "Database maintenance completed" \
+  --customAttributes '{cloud_vendor: "vendor_name", region: "us-east-1", isProd: true, instances: 2}' \
+  --validationFlags ALLOW_CUSTOM_CATEGORY_OR_TYPE,FAIL_ON_FIELD_LENGTH,FAIL_ON_REST_API_FAILURES \
+  --user "ops-team"
+
+# Custom category/type example using repeated --validationFlags style (each flag separately)
+newrelic changeTracking create \
+  --entitySearch "id = '<Entity GUID>'" \
+  --category Operational \
+  --type "Maintenance Window" \
+  --description "Database maintenance completed" \
+  --customAttributes '{cloud_vendor: "vendor_name", region: "us-east-1", isProd: true, instances: 2}' \
+  --validationFlags ALLOW_CUSTOM_CATEGORY_OR_TYPE \
+  --validationFlags FAIL_ON_FIELD_LENGTH \
+  --validationFlags FAIL_ON_REST_API_FAILURES \
+  --user "ops-team"`
+
+var CmdChangeTrackingCreate = &cobra.Command{
+	Use:   "create",
+	Short: "Create a New Relic change tracking event",
+	Long: `Create a New Relic change tracking event.
+
+This command allows you to create a change tracking event for a New Relic entity, supporting all fields in the Change Tracking GraphQL API schema for the changeTrackingCreateEvent mutation. For more information on each field, visit: https://docs.newrelic.com/docs/change-tracking/change-tracking-events/#change-tracking-event-mutation
+
+Required fields:
+  --entitySearch        Entity search query (e.g., name = 'MyService' AND type = 'SERVICE'). See our docs on 'entitySearch.query' under 'Required attributes' for more detailed examples: https://docs.newrelic.com/docs/change-tracking/change-tracking-events/#required-fields
+  --category            Category of event (e.g. Deployment, Feature Flag, Operational, etc.)
+  --type                Type of event (e.g. Basic, Rollback, Server Reboot, etc.)
+
+For Deployment events, the following are required/supported:
+  --version             Version of the deployment (required)
+  --changelog           Changelog for the deployment (URL or text)
+  --commit              Commit hash for the deployment
+  --deepLink            Deep link URL for the deployment
+
+For Feature Flag events, the following are required/supported:
+  --featureFlagId       ID of the feature flag (required)
+
+Other supported fields:
+  --description         Description of the event
+  --user                Username of the actor or bot
+  --groupId             String to correlate two or more events
+  --shortDescription    Short description for the event
+  --customAttributes    Custom attributes: use '-' for STDIN, '{...}' for inline JS object, or provide a file path
+  --validationFlags     Array of validation flags (e.g. ALLOW_CUSTOM_CATEGORY_OR_TYPE, FAIL_ON_FIELD_LENGTH, FAIL_ON_REST_API_FAILURES)
```

Comment:
I think "Array" is confusing here, since you can't pass in []. How about: 

```suggestion
  --validationFlags     Comma-separated list of validation flags (e.g. ALLOW_CUSTOM_CATEGORY_OR_TYPE, FAIL_ON_FIELD_LENGTH, FAIL_ON_REST_API_FAILURES)
```